.github/ISSUE_TEMPLATE/schema-change.md

```diff
@@ -0,0 +1,75 @@
+---
+name: Schema Change Checklist
+about: The checklist for making a schema change
+title: Schema Change: <changes>
+labels: ""
+assignees: ""
+---
+
+Welcome Change Captain! ⛵️
+
+When making schema changes, follow this guidance to avoid breaking changes:
+
+- Rather than modifying an existing field, consider adding a new field with a new name and later
+  deprecating the old field when it is no longer used
+- When deprecating an existing field, consider making it optional/nullable rather than removing it
+  altogether and then removing it entirely when all clients/active experiments no longer depend on
+  it
```

Comment:
Love this. 

When modifying a field by adding & deprecating, document:

* the expected behaviour of the old field in terms of the new.
* the expected behaviour when both fields are present. (e.g. in the presence of both fields, the new old field should be ignored).